### DISCUSSION TITLE
AppFlow cloud build fails

### DISCUSSION THREAD

**Discussion by dcesca** [rnmapbox/maps#3587] [Q&A]
Hi, I’m using AppFlow (https://ionic.io/appflow ) to build my react native projects but since it’s in cloud I cannot configure the netrc file.
Is there any other way I can pss the secret token so the cocoa pod install step won’t fail?

Thanks

![image](https://github.com/user-attachments/assets/ae7c7a23-6fbc-4d50-9bbe-e0ad7f044f7d)

**Comment by mfazekas** [comment]
See: `$RNMapboxMapsDownloadToken` 

https://github.com/rnmapbox/maps/blob/e51e63f519dd4ecbb992ebde206325d4e99dcca0/rnmapbox-maps.podspec#L195-L215
